extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -245,17 +305,21 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
             res.setHeader("Content-Type", "application/json");
             JSONWriter writer = new JSONWriter(res.getWriter());
             writer.object();
-            writer.key("code"); writer.value("ok");
-            if (!catalog.getBoolean("_exists")){
-                writer.key("catalogPost"); writer.value(catalogPost);
+            writer.key("code");
+            writer.value("ok");
+            if (!catalog.getBoolean("_exists")) {
+                writer.key("catalogPost");
+                writer.value(catalogPost);
             }
-            if (!dataset.getBoolean("_exists")){
-                writer.key("datasetPost"); writer.value(datasetPost);
+            if (!dataset.getBoolean("_exists")) {
+                writer.key("datasetPost");
+                writer.value(datasetPost);
             }
-            writer.key("distributionPost"); writer.value(distributionPost);
+            writer.key("distributionPost");
+            writer.value(distributionPost);
             writer.endObject();
 
-        }catch(Exception ex){
+        } catch (Exception ex) {
             respondException(res, ex);
         }
     }
```

Comment:
The implementation is 280 lines long, that's a bit much. Think about ways to make it smaller, and let's discuss that.